src/test/java/org/kohsuke/github/GHOrganizationTest.java

```diff
@@ -196,6 +197,30 @@ public void testCreateTeamWithRepoAccess() throws IOException {
         assertThat(team.getPermission(), equalTo(Permission.PUSH.toString().toLowerCase()));
     }
 
+    @Test
+    public void testCreateTeamWithRepoRole() throws IOException {
+        String REPO_NAME = "github-api";
+
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+        GHRepository repo = org.getRepository(REPO_NAME);
+
+        // Create team with access to repository. Check access was granted.
+        GHTeam team = org.createTeam(TEAM_NAME_CREATE).create();
+
+        RepositoryRole role = RepositoryRole.from(GHOrganization.Permission.TRIAGE);
+        team.add(repo, role);
+
+        // 'getPermission' does not return triage even though the UI shows that value
```

Comment:
@bitwiseman this has me pretty confused. the api doesn't return triage but the UI shows it being properly set.

i've been looking at the repository and team apis and i can't find anything.

<img width="890" alt="Screen Shot 2022-03-18 at 3 29 28 PM" src="https://user-images.githubusercontent.com/1831839/159088481-2f7adf31-ca36-4849-b47a-203540b30bad.png">

Reply:
Maybe it's a bug in GitHub?